### PR TITLE
[BIT] 转换能力补齐：broadcast_shape, dropout, dropout2d, dropout3d

### PR DESCRIPTION
**Pull request by Cutelemon6**
### paddle.broadcast_shape
转换规则为 result = torch.broadcast_shapes(x_shape, y_shape)，并且支持引擎解析 List[int] 的比较，其中有两条配置 torch 不支持对于 shape 中含有 -1 的推导，放入 paddle only

### paddle.nn.functional.dropout
随机函数，补齐转换能力后将配置放入 api_config_stochastic.txt

### paddle.nn.functional.dropout2d
随机函数，补齐转换能力后将配置放入 api_config_stochastic.txt

### paddle.nn.functional.dropout3d
随机函数，补齐转换能力后将配置放入 api_config_stochastic.txt

### CODE REVIEW

**Review by cszdrg — APPROVED** [review]

**Review by wanghuancoder — APPROVED** [review]
LGTM
